definitions/variable/tag_network_energy_carriers.yaml

```
@@ -0,0 +1,5 @@
+- Network-Based Energy Carriers:
+    - Natural Gas:
+        description: Natural gas
```

I suggest to stay in the "final energy" logic, because mainly final energy fuels are subject to transmission losses (e.g. district heat, electricity, gas) (could also bre Gas|Methane, but simply Gas is easier)
```suggestion
    - Gas:
        description: Gaseous energy carriers like fossil methane (Natural gas), biogenic methane, synthetic methane etc.
```